applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -67,21 +69,17 @@ static enum {
 	RUN_STATUS_MAX
 } run_status;
 
-/* global variable defined in different files */
-#if defined(CONFIG_SLM_NRF_CLOUD)
-extern bool nrf_cloud_ready;
-extern bool nrf_cloud_location_signify;
-#endif
 extern struct k_work_q slm_work_q;
 extern struct at_param_list at_param_list;
 extern uint8_t at_buf[SLM_AT_MAX_CMD_LEN];
 
-static int64_t utc_to_gps_sec(const int64_t utc_sec)
+static inline int64_t utc_to_gps_sec(const int64_t utc_sec)
 {
 	return (utc_sec - GPS_TO_UNIX_UTC_OFFSET_SECONDS) + GPS_TO_UTC_LEAP_SECONDS;
 }
 
-static void gps_sec_to_day_time(int64_t gps_sec, uint16_t *gps_day, uint32_t *gps_time_of_day)
+static inline void
+gps_sec_to_day_time(int64_t gps_sec, uint16_t *gps_day, uint32_t *gps_time_of_day)
 {
```

Comment:
This fucntion is only need under CONFIG_NRF_CLOUD_AGPS.
Please move to right above handle_at_agps().